app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -49,7 +44,9 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({ setFlowState
     checkRedirectResult();
   }, [setFlowState]);
 
-  async function handleSubmit({ email, name }: FormValues) {
+  async function handleSubmit(values: FormValues) {
```

Comment:
An update of this function to incorporate the new input fields will be handled by #307